packages/orchestration/src/orchestration-api.d.ts

```diff
@@ -0,0 +1,64 @@
+/**
+ * @file Public-facing API of orchestration.
+ * - must not depend on Cosmos types in anyway.
+ * - should remain relatively stable.
+ */
+import type { Amount, Brand } from '@agoric/ertp/exported.js';
+import type { IBCConnectionID } from '@agoric/vats';
+import type { LocalChainAccount } from '@agoric/vats/src/localchain.js';
+import type { ICQConnection } from './exos/icqConnectionKit.js';
+import type {
+  Chain,
+  ChainAmount,
+  ChainInfo,
+  Denom,
+  KnownChains,
+} from './types.js';
+
+/** Details for setup will be determined in the implementation. */
+export interface OrchestrationGovernor {
+  registerChain: (
+    chainName: string,
+    info: ChainInfo,
+    methods?: Record<string, any>,
+  ) => Promise<void>;
+}
+
+// chainName: managed like agoricNames. API consumers can make/provide their own
+export interface Orchestrator {
+  getChain: <C extends keyof KnownChains>(chainName: C) => Promise<Chain<C>>;
+
+  makeLocalAccount: () => Promise<LocalChainAccount>;
+  /** Send queries to ibc chains unknown to KnownChains */
+  provideICQConnection: (
```

Comment:
Why is this not just "query"?  Putting ICQ in the name is very specific and narrow (and doesn't account for e.g., v1 vs v2).

Reply:
It snuck in at some point. I suppose because https://github.com/agoric-labs/orchestration-api-spec/blob/main/src/orchestration.d.ts didn't have a query method. It's not implemented by anything so I'll remove it.